### ISSUE TITLE
[RC1] White screen after 2 fast taps on tab button

### ISSUE THREAD

**Issue by FilipSuk** [driftyco/ionic#8764]
#### Short description of the problem:
Situation: 3 tabs application where you have list of categories on the first tab. When you open one of the categories and then want to return back to the root with **two fast clicks** (one works ok) on first tab icon the first tab crashes and no pages are displayed - only white screen.

#### What behavior are you expecting?
Going back to root is expected.

**Steps to reproduce:**
1. Open (2nd level) page from root page of tabs application.
2. Tap quickly 2 times on the first tab icon
3. White screen

Repo: https://github.com/cernym43/ionic2-whiteScreen
Screencast: http://recordit.co/F6RKhju0Ul
![image](http://g.recordit.co/F6RKhju0Ul.gif)

**Solved relevant issue**: https://github.com/driftyco/ionic/issues/8319 - seems to be a wider issue of navigation.

My system information:

Cordova CLI: 6.1.1
Gulp version: CLI version 3.9.1
Gulp local: Local version 3.9.0
Ionic Framework Version: 2.0.0-rc.1
Ionic CLI Version: 2.1.1
Ionic App Lib Version: 2.1.1
Ionic App Scripts Version: 0.0.36
ios-deploy version: 1.9.0 
ios-sim version: Not installed
OS: Mac OS X El Capitan
Node Version: v6.8.1
Xcode version: Xcode 8.0 Build version 8A218a

Thank you for ionic!

**Comment by jgw96** [comment]
Hello, thanks for opening an issue with us! Unfortunately I am not able to reproduce this on my devices. What device or browser are you testing on?

**Comment by FilipSuk** [comment]
@jgw96 I also had hard time to reproduce it for the screencast, it doesn't happen every time. Try refreshing your browser and tapping bit slower, too fast taps does not reproduce issue (weird). Something between really fast and regular two taps reproduces the issue.

I was able to reproduce the bug in Chrome on Mac Air and on low end Android (Blackview BV2000).  Here is a screenshot how the DOM looks like after the "crash" (empty div):
![image](https://cloud.githubusercontent.com/assets/6044955/19455295/e6d51a90-94bc-11e6-8e26-d5f811d1085c.png)

I think the fix might be really similar to the one in other issue I referenced.
